p2p/raintree/router.go

```diff
@@ -0,0 +1,61 @@
+package raintree
+
+import (
+	"log"
+	"strings"
+)
+
+type router struct {
+	network *rainTreeNetwork
+}
+
+func newRouter(n *rainTreeNetwork) *router {
+	return &router{
+		network: n,
+	}
+}
+
+// GetTargetsAtLevel returns the targets for a given level
+func (r *router) GetTargetsAtLevel(level uint32) []target {
+	addrBookLenghtAtHeight := r.network.getAddrBookLengthAtHeight(level)
+	firstTarget := r.getTarget(firstMsgTargetPercentage, addrBookLenghtAtHeight, level)
```

Comment:
Please move the relevant constant from a `addrbook_utils` to this file where appropriate